### ISSUE TITLE
Add link to LinkedIn page

### ISSUE THREAD

**Issue by MarshalStewart** [Bearcat-Electric-Vehicle-BEV/BEV.github.io#17]
next to IG link

**Comment by MarshalStewart** [comment]
https://www.linkedin.com/company/bearcat-electric-racing/

**Comment by MarshalStewart** [comment]
Download linkedin icon
https://icons.getbootstrap.com/icons/linkedin/

**Comment by MarshalStewart** [comment]
<footer class="page-footer dark">
        <p class="text-center text-light">© 2021 Bearcat Electric Vehicle </p>
        <p class="text-center text-white-50" >
            <a href="https://www.instagram.com/bearcatev/" class="fa fa-instagram"></a>
        </p>
        <p class="text-center text-white-50" style="text-align: center;opacity: 0.53;"><em>Website Maintained by: </em><a href="https://www.linkedin.com/in/marshal-s-4a47b7126"><em>Marshal Stewart</em></a></p>
    </footer>
    
    Basically repeat the instagram code for linkedIn
